InternetHospital.BusinessLogic/Services/AppointmentService.cs

```diff
@@ -97,6 +100,99 @@ public IQueryable<AvailableAppointmentModel> GetAvailableAppointments(int doctor
             return appointments;
         }
 
+        /// <summary>
+        /// Delete doctor's appointment if it's not reserved
+        /// </summary>
+        /// <param name="appointmentId"></param>
+        /// <param name="doctorId"></param>
+        /// <returns></returns>
+        public (bool status, string message) DeleteAppointment(int appointmentId, int doctorId)
+        {
+            var appointment = _context.Appointments
+                .FirstOrDefault(a => a.Id == appointmentId);
+            if (appointment == null)
+            {
+                return (false, "Appointment not found");
```

Comment:
Back-end web api shouldn't return string messages visible to user. It's front-end job